frontend/src/api.ts

```diff
@@ -197,9 +200,15 @@ export const api = {
       authHeaders(token),
     );
   },
+
   async updateFact(token: string, id: number, data: IComponents["FactUpdate"]) {
     return axios.put(`${apiUrl}/api/facts/${id}`, data, authHeaders(token));
   },
+
+  async createMnemonic(token: string, data: IComponents["MnemonicCreate"]) {
```

Comment:
so createMnemonic is creating a 'log' of the mnemonic as opposed to having the user create the mnemonic itself? Probably good to edit the naming to reflect this

Reply:
Changed to `createMnemonicFeedbackLog`